Formula/haskell-language-server-wrapper.rb

```diff
@@ -0,0 +1,34 @@
+class HaskellLanguageServerWrapper < Formula
```

Comment:
Why do we still need the wrapper of everything is already in the HLS formula?

Reply:
Yup. Forgot about it 😅 Removed.